fedora_bot.py

```diff
@@ -97,6 +99,48 @@ def slack_notify(message: str):
     assert response.body == "ok"
 
 
+def get_open_pull_requests(args,component):
+    req = request.Request(f'https://src.fedoraproject.org/api/0/rpms/{component}/pull-requests', method="GET") # returns only open PRs by default
+    req.add_header('Content-Type', 'application/json')
+    req.add_header('Authorization', f'token {args.apikey}')
+
+    try:
+        r = request.urlopen(req)
+        content = r.read()
+        if content:
+            res = json.loads(content.decode('utf-8'))
+            if res['total_requests'] > 0:
+                i = 0
+                msg_info(f"Found {res['total_requests']} open pull requests for {component}. Starting the merge train...")
+                while i < res['total_requests']:
+                    merge_pull_request(args,component,res['requests'][i]['id'])
+                    i =+ 1
```

Comment:
Shouldn't this line be `i += 1`? Anyway, I think we should use for loop nevertheless, see above.